app/MMB/src/mmbList/CityImage.js

```diff
@@ -0,0 +1,122 @@
+// @flow
+
+import * as React from 'react';
+import {
+  StyleSheet,
+  NetworkImage,
+  Icon,
+  Color,
+  Text,
+  StretchedImage,
+  BlackToAlpha as gradient,
+} from '@kiwicom/mobile-shared';
+import { View } from 'react-native';
+import { DateFormatter, Translation } from '@kiwicom/mobile-localization';
+
+import TravelTypeIcon from './TravelTypeIcon';
+import ImageBadges from './ImageBadges';
+
+type Props = {|
+  imageUrl: string,
+  bookingId: string,
+  type: string,
+  departureCity: string,
+  arrivalCity: string,
+  date: Date,
+  status: string,
+|};
+
+export default class CityImage extends React.Component<Props> {
+  formatDate = () => {
+    const { date } = this.props;
+    return `${DateFormatter(date).format(
+      'ddd',
+    )} ${DateFormatter.getLocalizedWithoutYear(date)}`;
+  };
+
+  render = () => (
+    <View style={styles.container}>
+      <NetworkImage
+        source={{
+          uri: this.props.imageUrl,
+        }}
+        style={styles.image}
+        resizeMode="stretch"
```

Comment:
Is this really the best mode? It looks little bit funny on large/small devices. I would go for one with _"scale the image uniformly"_ (see: https://facebook.github.io/react-native/docs/image.html#resizemode).

Reply:
🙈